pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+## Requirements
+
+- Filling out the template is required. Any pull request that does not include enough information to be efficiently reviewed may be rejected.
+- Please consider which branch this is to be submitted against. This will normally be the development branch, so please ask your reviewer if you think it needs to go somewhere else. 
+
+## Type
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Improvement to existing feature
+
+## Related JIRA Issue(s)
+
+_Please provide the URL(s) for any JIRA issues related to this PR._
```

Comment:
The description of these sections can also be made as comments to make sure they don't end up in the PR description.

Reply:
That's done.

Reply:
Can you do this to the other sections as well?